src/events/cd-event-add-child-ticket.vue

```diff
@@ -185,6 +185,60 @@
     &__ticket-selector {
       max-width: 50%;
       padding: 8px 0px 24px;
-    }                   
+      min-height: 32px;
+    }
+     &__ticket-selector .multiselect {
+      font-family: Lato, Arial, Helvetica, sans-serif;
```

Comment:
Use the proper variable instead https://github.com/NateWr/bootstrap3/blob/master/bootstrap/less/variables.less#L45